src/main/java/executor/service/App.java

```diff
@@ -2,7 +2,7 @@
 
 import executor.service.config.di.FactoryForDI;
 import executor.service.config.di.ApplicationContext;
-import executor.service.service.ParalleFlowExecutorService;
+import executor.service.service.executionservice.ParalleFlowExecutorService;
```

Comment:
In my opinion, this is too strong a division into packages. I would leave it as is.